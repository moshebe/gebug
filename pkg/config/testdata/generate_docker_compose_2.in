name: my-app
output_binary: /app
build_command: go build -o {{.output_binary}}
runtime_image: golang:1.14
run_command: '{{.output_binary}}'
debugger_port: 40000
debugger_enabled: false
expose_ports:
  - 8080
networks:
  - frontend
  - backend