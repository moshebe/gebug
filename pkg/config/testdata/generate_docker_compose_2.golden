version: '3'
services:
  gebug-my-app:
    build:
      context: ..
      dockerfile: .gebug/Dockerfile
    volumes:
      - ../:/src:ro
    ports:
      - 8080
    networks:
      - frontend
      - backend
networks:
  frontend:
    external: true
  backend:
    external: true
